src/desktop/apps/auction_reaction/routes.jest.ts

```diff
@@ -0,0 +1,13 @@
+import { bidderRegistration } from "./routes"
+
+describe("Reaction Auction app routes", () => {
+  describe("bidderRegistration", () => {
+    it("defers handling using next() if the accepted-conditions=true query param is present", () => {
```

Comment:
is there any way to test that we _don't_ call `next()` if this query param if _not_ present. If so, I think that would be great from a code documentation perspective, particularly considering this trickiness